src/models/launch_1/README.md

```diff
@@ -16,7 +16,11 @@ The System76 Launch is a configurable keyboard with the following specifications
   - N-key rollover
 - Sockets and Switches
   - Kailh MX hot swap sockets
-  - Kailh Box Jade or Box Royal switches
+  - Default switch options:
+    - NovelKeys x Kailh Box Jade
```

Comment:
Why the callout that 2 of the options are "NovelKeys x Kailh Box XXX" keys?  I may make more sense to follow the Configure and Buy description and have the sound profile:
 * Kailh Box Jade - Clicky
 * Kailh Box royal - Tactile
 * Kailh Box Silent Pink - Silent Linear
 * Kailh Box Silent Brown - Silent

Reply:
It's explained in the commit message

Reply:
A web search for "Box Jade" or "Box Royal" show some websites simply referring to them as "NovelKeys Box Jade" / "NovelKeys Box Royal," so including NovelKeys in the specs here does make sense. At least a couple of other websites (KBDfans and Divinikey, from the first page of results) use the "NovelKeys x Kailh" format.

If we want to add the sound profile, I would recommend placing them in parenthesis instead of separating them with a hyphen, or having sub-bullets like `Sound Profile: Clicky` under each switch option.

Reply:
What I _really_ wanted to do was to link to a spec sheet for each switch, but Kailh's datasheet is pretty heavy, and Novelkeys's page on the switches are also product sales pages, not purely documentation pages. Regardless, it's pretty common for keyboards to simply list switch types without giving sound/tactility details, so I think it's fine the way it is.

Reply:
I'm going to resolve these switch comments if we're happy with it the way it is. Feel free to unresolve if I'm jumping the gun.